### ISSUE TITLE
Ultrasound Image recognition :Resnet50 Issue

### ISSUE THREAD

**Issue by shalini1153** [cv516Buaa/MMOTU_DS2Net#1]
Hello Team,

I am using the OTU_2d dataset to perform image recognition task (Task 3) and I am using ResNet50 to implement the model. I have resized the image to 384 x 384 and have used SGD with weight decay and momentum as 0.0005 and 0.9 .

But my accuracy is stuck at 50% , can you share across your model implementation for Task 3  or a link where I can find the code

**Comment by cv516Buaa** [comment]
Really happy that you follow our work. As for task 3, I suggest you to select mmclassification. For other repo, remember to use learning rate decay strategy.
